cmd/processManifest.go

```diff
@@ -12,20 +22,66 @@ var processManifestCmd = &cobra.Command{
 	Short: "Process a given manifest to meet the rhmi requirements.",
 	Long:  `Process a given manifest to meet the rhmi requirements.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("processImageManifests called")
+		//get csvfile
+		csv, filename, err := utils.ReadCSVFromBundleDirectory(manifestDir)
+		if err != nil {
+			handleError(err)
+		}
+		if filename == "" {
+			handleError(fmt.Errorf("No csv file found in the directory"))
+		}
+		fmt.Printf("Filename: %s", filename)
+		//populate a csv object from the file
+		filepath := fmt.Sprintf("%s/%s", manifestDir, filename)
+		fmt.Printf("Filepath: %s", filepath)
+		err = utils.PopulateObjectFromYAML(filepath, csv)
+		if err != nil {
+			handleError(err)
+		}
+		// make the updates to that file
+		// TODO Get the correct replaces value and update it.
+		csv.Spec.Replaces = "Some Replaces"
+		updateEnvs(csv.Spec.InstallStrategy.StrategySpec.DeploymentSpecs)
+		//write the file out using the object
+		err = utils.WriteObjectToYAML(csv, filepath)
+		if err != nil {
+			handleError(err)
+		}
 	},
 }
 
 func init() {
 	ewsCmd.AddCommand(processManifestCmd)
 
-	// Here you will define your flags and configuration settings.
-
-	// Cobra supports Persistent Flags which will work for this command
-	// and all subcommands, e.g.:
-	// processManifestCmd.PersistentFlags().String("foo", "", "A help for foo")
+	processManifestCmd.Flags().StringVarP(&manifestDir, "manifest-dir", "m", "", "Manifest Directory Location.")
```

Comment:
Went back and forth on the naming of this on my pr, i currently have src-dir and dest-dir, but maybe i should revert back to src-manifest-dir and dest-manifest-dir for consistency :thinking: